### PR TITLE
Anaconda: set base environment deactivated

### PR DESCRIPTION
**Pull request by core-man**

### CODE REVIEW

File: source/anaconda/index.rst [hunk 1]
```diff
@@ -95,6 +95,12 @@ Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包、管理虚
 
    $ conda deactivate
 
+.. note::
+
+   安装好 Anaconda 后，打开终端默认会激活 base 环境，可以通过如下命令取消此设置::
+   
+      conda config --set auto_activate_base False or True
```
Comment (core-man) [inline source/anaconda/index.rst:102]:
```suggestion
      conda config --set auto_activate_base False
```

File: source/anaconda/index.rst [hunk 2]
```diff
@@ -95,6 +95,12 @@ Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包、管理虚
 
    $ conda deactivate
 
+.. note::
+
+   安装好 Anaconda 后，打开终端默认会激活 base 环境，可以通过如下命令取消此设置::
```
Comment (core-man) [inline source/anaconda/index.rst:100]:
```suggestion
   安装 Anaconda 后，打开终端默认会激活 base 环境，可以通过如下命令取消此设置::
```

File: source/anaconda/index.rst [hunk 3]
```diff
@@ -95,6 +95,12 @@ Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包、管理虚
 
    $ conda deactivate
 
+.. note::
```
Comment (seisman) [inline source/anaconda/index.rst:98]: Why do you want to disable the base environment?

Comment (core-man) [reply source/anaconda/index.rst:98]: Why do you want to activate it?

Comment (core-man) [reply source/anaconda/index.rst:98]: Sometimes when I don't want to use python and open a shell, but there is `base` in the console.

Comment (seisman) [reply source/anaconda/index.rst:98]: https://stackoverflow.com/questions/54429210/how-do-i-prevent-conda-from-activating-the-base-environment-by-default

A related question.

Comment (core-man) [reply source/anaconda/index.rst:98]: done

File: source/anaconda/index.rst [hunk 4]
```diff
@@ -95,6 +95,17 @@ Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包、管理虚
...
+
+   安装 Anaconda 后，打开终端默认会激活 base 环境，可以通过如
+   下命令取消或激活此默认设置::
+   
+      conda config --set auto_activate_base False
+      conda config --set auto_activate_base True
+
+   取消此设置后，使用如下命令可临时激活 base 环境::
+      $ conda activate base
+
```
Comment (core-man) [inline source/anaconda/index.rst:108]:
```suggestion
   安装 Anaconda 后，打开终端默认会激活 base 环境。不经常使用
   Python 的读者可以通过如下命令取消此默认设置::
   
      $ conda config --set auto_activate_base False
      
   取消后，可以临时激活 base 环境::
      $ conda activate base
      
   重新激活此默认设置::
   
      $ conda config --set auto_activate_base True

```

**Review by seisman — APPROVED** [review]

File: source/anaconda/index.rst [hunk 5]
```diff
@@ -95,6 +95,20 @@ Anaconda 中提供的 ``conda`` 命令可以用于安装 Python 包、管理虚
...
+.. note::
+
+   安装 Anaconda 后，打开终端默认会激活 base 环境。不经常使用
+   Python 的读者可以通过如下命令取消此默认设置::
+   
+      $ conda config --set auto_activate_base False
+      
+   取消后，可以临时激活 base 环境::
+      $ conda activate base
+
```
Comment (core-man) [inline source/anaconda/index.rst:107]:
```suggestion
      
   取消后，可以临时激活 base 环境::

      $ conda activate base
      
```

**Pull request merged**